build/images/Dockerfile.clickhouse.monitor.ubuntu

```diff
@@ -0,0 +1,19 @@
+ARG GO_VERSION
```

Comment:
nit: maybe rename this file `Dockerfile.clickhouse-monitor.ubuntu`

Reply:
Renamed and added the folder.